crates/tests/mqtt_tests/src/test_mqtt_server.rs

```diff
@@ -86,69 +89,76 @@ fn spawn_broker(port: u16) {
     });
 
     std::thread::spawn(move || {
-        let mut rx = tx.connect(200).unwrap();
         tx.subscribe("#").unwrap();
 
         loop {
-            if let Some(message) = rx.recv().unwrap() {
-                for chunk in message.payload.into_iter() {
-                    let mut bytes: Vec<u8> = vec![];
-                    for byte in chunk.into_iter() {
-                        bytes.push(byte);
-                    }
-                    let payload = match std::str::from_utf8(bytes.as_ref()) {
+            let notification = match rx.recv().unwrap() {
+                Some(v) => v,
+                None => continue,
+            };
+            match notification {
+                Notification::Forward(forward) => {
+                    let payload = match std::str::from_utf8(&forward.publish.payload) {
                         Ok(payload) => format!("{:.110}", payload),
-                        Err(_) => format!("Non uft8 ({} bytes)", bytes.len()),
+                        Err(_) => format!("Non uft8 ({} bytes)", forward.publish.payload.len()),
                     };
                     eprintln!(
-                        "MQTT-TEST MSG: topic = {}, payload = {:?}",
-                        message.topic, payload
+                        "MQTT-TEST MSG: topic = {:?}, payload = {:?}",
+                        forward.publish.topic, payload
                     );
                 }
+
+                v => {
+                    println!("{:?}", v);
+                }
             }
         }
     });
 }
 
 fn get_rumqttd_config(port: u16) -> Config {
-    let router_config = librumqttd::rumqttlog::Config {
-        id: 0,
-        dir: "/tmp/rumqttd".into(),
-        max_segment_size: 10240,
-        max_segment_count: 10,
-        max_connections: 10,
+    let console_settings = {
+        let mut conset: ConsoleSettings = Default::default();
+        conset.listen =
+            SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::new(0, 0, 0, 0), 3030)).to_string();
+        conset
+    };
+
+    let rf = RouterConfig {
+        instant_ack: false,
+        max_segment_size: 104857600,
+        max_segment_count: 100,
+        max_read_len: 10240,
+        max_connections: 5000,
+        initialized_filters: None,
     };
 
     let connections_settings = ConnectionSettings {
-        connection_timeout_ms: 10,
-        max_client_id_len: 256,
+        connection_timeout_ms: 60000,
         throttle_delay_ms: 0,
         max_payload_size: 268435455,
         max_inflight_count: 200,
-        max_inflight_size: 1024,
-        login_credentials: None,
+        max_inflight_size: 10204,
+        dynamic_filters: false,
     };
 
     let server_config = ServerSettings {
         listen: SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::new(0, 0, 0, 0), port)),
-        cert: None,
-        next_connection_delay_ms: 1,
+        next_connection_delay_ms: 0,
```

Comment:
Why was it zero earlier and why change that to 1 now? What's the impact?

Reply:
reverted back